tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -238,6 +240,50 @@ def _generate_op_table(self, group_by_input_shape=False):
         data = {"data": {"columns": columns, "rows": rows}}
         return data
 
+    def _generate_op_stack_calls(self, group_by_input_shape=False):
```

Comment:
refactor the code with _generate_op_table to extract the stack_lists_dict and call_stack?

Reply:
Thanks, done.